core/src/test/java/apoc/load/XmlTest.java

```diff
@@ -304,36 +288,6 @@ private void commonAssertionsWithNextWordRels(String query, Map<String, Object>
                 });
     }
 
-
-    @Test
-    public void testLoadXmlWithNextEntityRels() {
-        testCall(db, "call apoc.xml.import('file:" + FILE_SHORTENED + "', " +
-                        "{connectCharacters: true, filterLeadingWhitespace: true}) yield node",
-                row -> assertNotNull(row.get("node")));
-        testResult(db, "match (n) return labels(n)[0] as label, count(*) as count", result -> {
-            final Map<String, Long> resultMap = result.stream().collect(Collectors.toMap(o -> (String)o.get("label"), o -> (Long)o.get("count")));
-            assertEquals(2L, (long)resultMap.get("XmlProcessingInstruction"));
-            assertEquals(1L, (long)resultMap.get("XmlDocument"));
-            assertEquals(369L, (long)resultMap.get("XmlCharacters"));
-            assertEquals(158L, (long)resultMap.get("XmlTag"));
-        });
-
-        // no node more than one NEXT/NEXT_SIBLING
-        testCallEmpty(db, "match (n) where size([p =  (n)-[:NEXT]->()  | p ]) > 1 return n", null);
-        testCallEmpty(db, "match (n) where size([p =  (n)-[:NEXT_SIBLING]->()  | p ]) > 1 return n", null);
-
-        // no node more than one IS_FIRST_CHILD / IS_LAST_CHILD
-        testCallEmpty(db, "match (n) where size([p =  (n)<-[:FIRST_CHILD_OF]-()  | p ]) > 1 return n", null);
-        testCallEmpty(db, "match (n) where size([p =  (n)<-[:LAST_CHILD_OF]-()  | p ]) > 1 return n", null);
-
-        // NEXT_WORD relationship do connect all word nodes
-        testResult(db, "match p=(:XmlDocument)-[:NE*]->(e:XmlCharacters) where not (e)-[:NE]->() return length(p) as len",
-                result -> {
-                    Map<String, Object> r = Iterators.single(result);
-                    assertEquals(369L, r.get("len"));
-                });
-    }
-
```

Comment:
Same type of comment of having no tests for apoc.import.xml which is replacing this.